extensions/data-transfer/portability-data-transfer-google/src/test/java/org/datatransferproject/datatransfer/google/mediaModels/GoogleMediaItemTest.java

```diff
@@ -84,81 +86,101 @@ public void getMimeType_videoModel_mimeTypeFromFilename() {
     for (Entry entry: filenameToMimeTypeMap.entrySet()) {
       videoMediaItem.setMimeType("INVALID_MIME");
       videoMediaItem.setFilename(entry.getKey().toString());
+       VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(),
+            videoMediaItem);
 
-      VideoModel videoModel = GoogleMediaItem.convertToVideoModel(Optional.empty(), videoMediaItem);
-
-      assertEquals(entry.getValue(), videoModel.getMimeType());
+        assertEquals(entry.getValue(), videoModel.getMimeType());
```

Comment:
are these supposed to be extra indented? or is it something happening on my end

Reply:
Fixed.